pennylane/qinfo/transforms.py

```diff
@@ -645,11 +645,11 @@ def circuit1():
 
     def evaluate_fidelity(all_args0=None, all_args1=None):
         """Wrapper used for evaluation of the fidelity between two states computed from QNodes. It allows giving
-        the args and kwargs to each :class:`.QNode`.
+        the args to each :class:`.QNode`.
 
         Args:
-            all_args0 (tuple): Tuple containing the arguments (*args, **kwargs) of the first :class:`.QNode`.
-            all_args1 (tuple): Tuple containing the arguments (*args, **kwargs) of the second :class:`.QNode`.
+            all_args0 (tuple): Tuple containing the arguments (*args) of the first :class:`.QNode`.
+            all_args1 (tuple): Tuple containing the arguments (*args) of the second :class:`.QNode`.
```

Comment:
Thank you for pointing this out! Could you remove this part; it is fixed [here](https://github.com/PennyLaneAI/pennylane/pull/2740)